src/install-protoc.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly PROTOC_VER="3.12.3"
+readonly PROTOC_ARCHIVE="protoc-${PROTOC_VER}-linux-$( uname -m ).zip"
+readonly PROTOC_URL="https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VER}/${PROTOC_ARCHIVE}"
+
+readonly PROTOC_GEN_GO_VER="1.25.0"
+readonly PROTOC_GEN_GO_ARCHIVE="protoc-gen-go.v${PROTOC_GEN_GO_VER}.linux.amd64.tar.gz"
+readonly PROTOC_GEN_GO_URL="https://github.com/protocolbuffers/protobuf-go/releases/download/v${PROTOC_GEN_GO_VER}/${PROTOC_GEN_GO_ARCHIVE}"
```

Comment:
This is the main change in this PR - previously https://github.com/golang/protobuf/tree/master/protoc-gen-go was used